independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/gitlab-ci/base/.gitlab-ci.tpl.qute.yml

```diff
@@ -0,0 +1,141 @@
+#
+# Template GitLab CI to build jvm, native and native-distroless images.
```

Comment:
we really need to find a way to parameterize what approach is taken here as distroless isn't the only approach.